.github/checkgroup.yml

```diff
@@ -154,15 +154,30 @@ subprojects:
       - "setup.cfg"  # includes pytest config
       - ".actions/**"
     checks:
-      - "lite-cpu (macOS-11, lite, 3.10, latest, stable)"
       - "lite-cpu (macOS-11, lite, 3.7, latest, stable)"
       - "lite-cpu (macOS-11, lite, 3.7, oldest, stable)"
-      - "lite-cpu (ubuntu-20.04, lite, 3.10, latest, stable)"
+      - "lite-cpu (macOS-11, lite, 3.8, latest, stable)"
```

Comment:
Can we rather have a full matrix with some exclude, this is hard to read and when we add parameter `lightning` it will be just double